moveit_ros/planning/rdf_loader/src/synchronized_string_parameter.cpp

```diff
@@ -123,19 +127,17 @@ bool SynchronizedStringParameter::waitForMessage(const rclcpp::Duration timeout)
       name_, rclcpp::QoS(1).transient_local().reliable(),
       std::bind(&SynchronizedStringParameter::stringCallback, this, std::placeholders::_1));
 
-  rclcpp::WaitSet wait_set;
-  wait_set.add_subscription(string_subscriber_);
+  auto timeout_ns =
+      std::chrono::duration_cast<std::chrono::nanoseconds>(timeout.to_chrono<std::chrono::duration<double>>());
```

Comment:
Can this be simplified to just this:

```suggestion
  auto timeout_ns =
      timeout.to_chrono<std::chrono::duration<std::chrono::nanoseconds>>();
```

Reply:
Or maybe even simpler (idk if this works):

```suggestion
  auto timeout_ns = timeout.to_chrono<std::chrono::nanoseconds>();
```